.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
What is your reasoning for configuring both renovate and dependabot?
I usually just enable security updates by depandabot in the repo setting and use renovate for the rest, because I find some behavior of dependabot really annoying.
But if you have a string opinion about using depandabot, feel free to drop the renovate.json file.